boards/nxp/ls1046ardb/board.yml

```diff
@@ -0,0 +1,8 @@
+board:
+  name: nxp_ls1046ardb
+  vendor: nxp
+  socs:
+  - name: ls1046a
+    variants:
+    - name: smp_2cores
+    - name: smp_4cores
```

Comment:
non-blocking but in other boards smp is a variant then 2cores and 4cores are variants of that